docs/en/operation/Admin-CLI.md

```diff
@@ -70,7 +71,12 @@ $ ./bin/alluxio fsadmin journal quorum info -domain <MASTER | JOB_MASTER>
 
 ```console
 # Remove a member from leader election quorum.
-$ ./bin/alluxio fsadmin journal quorum remove -domain <MASTER | JOB_MASTER> -address <Member_Address>
+$ ./bin/alluxio fsadmin journal quorum remove -domain <MASTER | JOB_MASTER> -address <HostName:Port>
```

Comment:
nit: change `<HostName:Port>` to `<HOSTNAME:PORT>`
same thing down a few lines

Reply:
fixed